mantidproject/mantid
# Bin MD dialog crash

resolved unprotected use of unititialised pointer
Also added missing usage examples for BinMD and CreateMDWrokspace

re #13039

This was an error introduced as part of another change -> not one for the release notes
### to test
1. Start Mantidplot, run this script

``` python
mdws = CreateMDWorkspace(Dimensions=3, Extents='-10,10,-10,10,-10,10', Names='A,B,C', Units='U,U,U')
FakeMDEventData(InputWorkspace=mdws, PeakParams='500000,0,0,0,3')
```
1. start the binMD dialog, select mdws as the input workspace and adccept the defaults elsewhere.
2. Execute -> success!
3. Also check the new usage examples for BinMD and CreateMDWrokspace
